docker/dockerfiles/Dockerfile

```diff
@@ -133,6 +133,7 @@ RUN echo 'eval "$(rbenv init -)"' >> ~/.bashrc
 # We need git >= 2.15 to use git rev-parse --is-shallow-clone feature
 # TODO: consolidate apt-get installs
 RUN sudo apt-get install -y software-properties-common && sudo add-apt-repository ppa:git-core/ppa && sudo apt-get update && sudo apt-get install -y git
+RUN sudo apt-get install -y jq
```

Comment:
NIT: Does it make sense to include this in the previous line?

Reply:
It does but when I rebuilt I wanted to reuse the existing build cache to make it faster 😅

I have a TODO to consolidate these installs with the ones at the top of the file once things are stable.